src/Framework/OpenXmlBasePart.cs

```diff
@@ -81,6 +81,11 @@ internal void Load(OpenXmlPackage openXmlPackage, OpenXmlPart parent, Uri uriTar
             this._uri = uriTarget;
 
             // TODO: should we delay load?
+            if (!this.OpenXmlPackage.Package.PartExists(uriTarget))
+            {
+                throw new OpenXmlPackageException(string.Format("The part with URI target {0} does not exist in the package.", uriTarget));
+            }
```

Comment:
OK, so the repro is open a PDF in Adobe Acrobat 10.0.0 and save it as an MS Word file. Acrobat messes up when it detects words it thinks it can OCR (like header images) etc. Opening the file in Microsoft Word will work fine, but the SDK will fail. The workaround for the users was to open the file in MS Word, make a minor change, and hit save, MS Word would fix up the fault. Example public files can be downloaded from: https://drive.google.com/folderview?id=0By6nm3ACZguvfnJlRWdsWWtUTTN1dkh4cnVXbkhXVnpaRXdwQS0tejNOREVlTDdrem8tYWM&usp=sharing